automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestModelDescrs.scala

```diff
@@ -0,0 +1,33 @@
+package org.clulab.aske.automates.text
+
+import org.clulab.aske.automates.TestUtils._
+
+class TestModelDescrs extends ExtractionTest{
+
+  val t1a = "SWAT incorporates a simple empirical model to predict the trophic status of water bodies."
```

Comment:
I got rid of 'the' in the phrase because that's what we do with most other events. I wonder if there is a way to apply coref resolver here as well? Like if model name contains a certain list of generic words like simple, big, small, and such, resolve it to a model name within the sentence. Let'd discuss tomorrow. Fine for this pr. Also might be fine to include that extractions in the test.

Reply:
Yes, determiners are not needed here. Thank you!

Reply:
And coref resolver might be applicable here, I'm not sure though. But that is an interesting idea! Let's discuss this in our meeting tomorrow.